From e4c47fc7552a0c3021b62e132c353e2fcd4d4a17 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Thu, 29 Oct 2020 09:25:20 +0800
Subject: [PATCH] vo_gpu: hwdec_drmprime_drm: Change video plane zpos to 1

The behavior of having same zpos with primary plane is undefined.

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 video/out/opengl/hwdec_drmprime_drm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/video/out/opengl/hwdec_drmprime_drm.c b/video/out/opengl/hwdec_drmprime_drm.c
index 329cbccb99b..3682df27dea 100644
--- a/video/out/opengl/hwdec_drmprime_drm.c
+++ b/video/out/opengl/hwdec_drmprime_drm.c
@@ -204,7 +204,7 @@ static int overlay_frame(struct ra_hwdec *hw, struct mp_image *hw_image,
                 drm_object_set_property(request, p->ctx->drmprime_video_plane, "CRTC_Y",  MP_ALIGN_DOWN(dy, 2));
                 drm_object_set_property(request, p->ctx->drmprime_video_plane, "CRTC_W",  dstw);
                 drm_object_set_property(request, p->ctx->drmprime_video_plane, "CRTC_H",  dsth);
-                drm_object_set_property(request, p->ctx->drmprime_video_plane, "ZPOS",    0);
+                drm_object_set_property(request, p->ctx->drmprime_video_plane, "ZPOS",    1);
             } else {
                 ret = drmModeSetPlane(p->ctx->fd, p->ctx->drmprime_video_plane->id, p->ctx->crtc->id, next_frame.fb.fb_id, 0,
                                       MP_ALIGN_DOWN(dx, 2), MP_ALIGN_DOWN(dy, 2), dstw, dsth,

